metacat-common/src/main/java/com/netflix/metacat/common/dto/notifications/sns/messages/RenameTableMessage.java

```diff
@@ -51,8 +55,10 @@ public RenameTableMessage(
         @JsonProperty("timestamp") final long timestamp,
         @JsonProperty("requestId") final String requestId,
         @JsonProperty("name") final String name,
-        @JsonProperty("payload") final UpdatePayload<TableDto> payload
+        @JsonProperty("newName") final String newName,
+        @Nullable @JsonProperty("payload") final UpdatePayload<TableDto> payload
```

Comment:
Wondering why do we want to have the payload be Nullable?

Reply:
In case the initial send fails, for example if full payload causes message to be too long, then the fallback is send the message again except null payload.